orc8r/gateway/python/magma/common/service_registry.py

```diff
@@ -85,6 +86,31 @@ def reset():
         """
         ServiceRegistry.get_registry()["services"] = {}
 
+    @staticmethod
+    def get_bare_bootstrap_rpc_channel(domain_name, port, rootca_path=None):
+        """
+        Return an RPC channel to bootstrap service in CLOUD, without referencing a
+        control proxy config.
+
+        Returns:
+            grpc channel
+        """
+        rootca_path = ROOTCA_PATH if not rootca_path else rootca_path
```

Comment:
I believe you can do

```suggestion
        rootca_path = rootca_path or ROOTCA_PATH
```